src/NuGet.Clients/NuGet.CommandLine/Commands/DownloadCommandBase.cs

```diff
@@ -38,6 +38,9 @@ public ICollection<string> Source
         [Option(typeof(NuGetCommand), "CommandNoCache")]
         public bool NoCache { get; set; }
```

Comment:
This PR does not affect the behavior of the -NoCache option. The bug noted in NuGet/Home#1406 that -NoCache option is ignored still remains.